src/include/cortxfs.h

```diff
@@ -107,6 +107,13 @@ int cfs_endpoint_scan(int (*cfs_scan_cb)(const struct cfs_endpoint_info *info,
 #define CFS_DEFAULT_CONFIG "/etc/cortx/cortxfs.conf"
 
 #define CFS_ROOT_INODE 2LL
+/*
+ * ref. m0_lid_to_unit_map[], we will support block sizes from 4K till 1M.
```

Comment:
Suggestion: There should not be any motr references in cortx fs since cortx-fs can work on different backends (theoretically) . We can say that we support block sizes from 4K to 1M in multiples of 4k.

Reply:
Ok will remove the comment. motr's values are not exact multiples of 4K, e.g. 12288 is not there in motr's array, values are 2^12 to 2^20.